lib/Lib.IntTypes.fst

```diff
@@ -131,6 +249,11 @@ let add_mod #t #l a b =
   | U32  -> UInt32.add_mod a b
   | U64  -> UInt64.add_mod a b
   | U128 -> UInt128.add_mod a b
+  | S8   -> Int8.int_to_t (Int.add_mod (Int8.v a) (Int8.v b))
```

Comment:
Removing these as discussed, but note that this wouldn't extract because it uses mathematical integers